asmcli/tests/run_basic_suite_managed

```diff
@@ -15,7 +15,7 @@ main() {
   # CLI setup
   parse_args "$@"
 
-  run_basic_test "install" "mesh_ca" "--managed"; RETVAL=$?;
+  run_basic_test "install" "mesh_ca" "--managed -o cni-managed"; RETVAL=$?;
```

Comment:
Please verify the file path. Looks like its erroring out because it tried to use absolute path maybe?

`asmcli: Running: '/usr/bin/kubectl --kubeconfig /tmp/tmp.tHmcwE8h75 --context gke_asm-scriptaro-oss_us-central1-c_long-term-test-cluster apply --overwrite=true -f /cni-managed.yaml'
asmcli: -------------
error: the path "/cni-managed.yaml" does not exist`